massa-db-worker/src/lib.rs

```diff
@@ -1,3 +1,51 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+//! # General description
+//!
+//! MassaDB is a wrapper around:
+//! * A RocksDB database (on Disk)
+//! * Some caches (on RAM).
+//! * a config
+//!
+//! # RocksDB
+//!
+//! RocksDB stores keys and values, which are arbitrarily-sized byte streams (aka vec<u8> or &[u8]).
+//! It supports both point lookups and range scans.
+//!
+//! For MassaDB, we use 3 rocksdb column:
+//! * state: all data used to compute the db hash (or final state hash)
+//! * versioning: partial MIP store data see Versioning doc section: "MipStore and Final state hash"
+//! * metadata: final state hash + slot
+//!
+//! Note that data is stored with a prefix (see constants.rs in massa-db-exports).
+//! For instance, a ledger update, will be stored (in column: 'state') as:
+//! * key: LEDGER_PREFIX+Address+[...] (serialized as bytes)
+//! * value: Amount (serialized as bytes)
```

Comment:
Think it would be nice to explain that final state information is all stored at the same level with prefixes working as a way to individual parts of data together. Here you talk about prefixes but don't explain why they are there.